index.html

```diff
@@ -609,12 +609,10 @@ <h2>
           member is set.
         </p>
       </div>
-      <p data-link-for="PaymentRequest">
-        Because the simultaneous display of multiple <a>PaymentRequest</a> user
-        interfaces might confuse the user, this specification limits the
-        <a>user agent</a> to displaying one at a time via the <a>show()</a>
-        method. This is ensured by a <dfn>payment request is showing</dfn>
-        boolean.
+      <p>
+        Each <a>top-level browsing context</a> has a <dfn>payment request is
+        showing</dfn> boolean, which prevents showing more than one payment UI
+        at a time.
```

Comment:
Hi @marcoscaceres,

Editorial suggestion for the above 

        ....showing</dfn> boolean to support <a>user agent</a> management of payment UI display. For example,
       a <a>user agent</a> might choose to show only one payment UI at a time.

Reply:
Appreciate this suggestion, but it doesn't quite capture what we are trying to restrict here (don't show two payment UI in the same browser tab). I'll add a note describing what you are saying above.